manifest.json

```diff
@@ -6,15 +6,15 @@
         "en": "Manage PostgreSQL databases over the web",
         "fr": "Application web de gestion des bases de données PostgreSQL"
     },
-    "version": "4-5.7~ynh1",
+    "version": "5.7~ynh1",
```

Comment:
Well, the real version is 4-5.7 not 5.7 so we can't just remove this. The main version of pgadmin is "pgadmin 4"

Reply:
I made a issue about this problem here: https://github.com/YunoHost/package_linter/issues/102

Reply:
fair enough.